n2kMappings.js

```diff
@@ -148,7 +184,196 @@ exports.mappings = {
     filter: function(n2k) {
       return n2k.fields['Engine Instance'] === 'Dual Engine Starboard';
     }
-  }],
+  }, {
+    source: 'Oil pressure',
+    node: 'propulsion.port.oilPressure',
+    filter: function(n2k) {
+      return n2k.fields['Engine Instance'] === 'Single Engine or Dual Engine Port';
+    }
+  }, {
+    source: 'Total Engine hours',
+    node: 'propulsion.port.runTime',
+    filter: function(n2k) {
+      return n2k.fields['Engine Instance'] === 'Single Engine or Dual Engine Port';
+    }
+  }, 
+   {
+    node: 'notifications.checkEngine',
```

Comment:
This engine notifications code would a bit less unwieldy if implemented as  lookup table, see https://github.com/sbender9/n2k-signalk/blob/9e9916f216ff3ef47399435ca1e6e7e06c02733e/n2kMappings.js#L669